e2e/schedule/sdk.config.toml

```diff
@@ -5,7 +5,7 @@ confirmations = 3
 
 ckb_rpc_url = "https://testnet.ckbapp.dev/"
 
-[user_lock_script]
+[module_lock_script]
```

Comment:
this toml key has changed?

Reply:
See https://github.com/synapseweb3/forcerelay-ckb-sdk/pull/31